dev-libs/libxslt/libxslt-1.1.38.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Note: Please bump this in sync with dev-libs/libxml2.
+
+PYTHON_COMPAT=( python3_{9..11} )
```

Comment:
please drop 9 on bumps (write it as 10..11 to make it easier for 3.12)

Reply:
Corrected in https://github.com/gentoo/gentoo/commit/84fbee5f6c36ff1057e2ca04e583f0e1305882d9 , Thanks!